deploy_env.sh

```diff
@@ -198,10 +273,6 @@ fi
 
 source ${V_ENV_DIR}/bin/activate
 pip install -U pip
-pip install git+https://github.com/openstack/fuel-devops.git@${FUEL_DEV_VER} --upgrade
```

Comment:
There is `dos.py` call in code below (but before installing fuel-qa requirements)